Plugins/Flow.Launcher.Plugin.Shell/Main.cs

```diff
@@ -317,7 +317,7 @@ public void Init(PluginInitContext context)
 
         bool API_GlobalKeyboardEvent(int keyevent, int vkcode, SpecialKeyState state)
         {
-            if (_settings.ReplaceWinR)
+            if (!context.CurrentPluginMetadata.Disabled && _settings.ReplaceWinR)
```

Comment:
Is it easy to full disable the plugin without doing it here? Obviously taking into consideration that disabled plugin still needs to be shown in the Settings window

Reply:
Not easy due to the current model. We cannot make sure the plugin won't do anything or register anything under its constructor or Init method, so we can only assume that.